backend/msgpack/conn_test.go

```diff
@@ -134,7 +134,7 @@ func TestConnectionReconnectProperties(t *testing.T) {
 	props := getProperties()
 	props.Property(
 		`When the attempt to reconnect fails writes should:
```

Comment:
nit: I know this isn't your change, but perhaps we should clarify this to
s/When the attempt to reconnect fails writes should/When there is no active connection and a write cannot establish one it should:

Reply:
Sounds good.